### ISSUE TITLE
Keyboard

### ISSUE THREAD

**Issue by zeldaking** [libretro/citra#108]
I couldnt find anyone talking about this but when there is usually a keyboard prompt, it just puts "CITRA". Is there a fix for this? Im pretty sure its because the citra core is old.

**Comment by zeldaking** [comment]
Also gamecube crashes

**Comment by Haelikor** [comment]
I managed to get Mii Maker working, and dump my Mii data into Libretro, but it looks like some 3ds games use their own keyboard and work fine (SMT IV) while a lot (example: Legend of Zelda Link Between Worlds) call some kind of system keyboard that fails. Seems to be hardcoded. 

Only way I could bypass the problem: run stand-alone Citra, change my config there to have my name, make a save, copy save over. You can add game directories in stand-alone Citra, right click the game, and have its save folder pop up. Path will be the same in RA so from there, it's just copy-paste.

Hopefully they fix this down the road.

**Comment by ferrazpedro** [comment]
Yeah, some games like Digimon World Re:Digitize Decode become kinda bothersome to play since you have to give names to your monsters all the time and this always happens, it just don't open any input prompt and just fills it as CITRA...
